homeassistant/components/nws/const.py

```diff
@@ -26,6 +26,7 @@
 ATTRIBUTION = "Data from National Weather Service/NOAA"
 
 ATTR_FORECAST_DETAILED_DESCRIPTION = "detailed_description"
+ATTR_FORECAST_SUMMARY = "short_forecast"
```

Comment:
I'm open to it.  For normalization of variables, etc. it would make sense.  It's easy enough to make the change, and I suspect there are only a couple of folks who are doing the custom_component thing to expose the short forecast at the moment.  Let me know one way or the other and I'll push an updated commit to this PR.  Thanks again.

Reply:
I think we should be consistent and use `short_description`, it is too late to go back and decide if we should have used `detailed_forecast` instead.

Reply:
Sounds good to me.